File: yao_node.cpp
#include<ros/ros.h>
#include<std_msgs/String.h>

int main(int argc,char *argv[])
{
    ros::init(argc,argv,"yao_node");
    printf("过去生于未来！\n");

    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("ge_ge_dai_wo",10);

    ros::Rate loop_rate(10);

    while(ros::ok())
    {
        printf("我要开始刷屏了!\n");
        std_msgs::String msg;
        msg.data="求上车++++";
        pub.publish(msg);
        loop_rate.sleep();
    }
    return 0;
}

File: yao_node.py
#!/usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String

if __name__=="__main__":
    rospy.init_node("yao_node")
    rospy.logwarn("过去生于未来！")

    pub = rospy.Publisher("ge_ge_dai_wo",String,queue_size=10)

    rate = rospy.Rate(10)

    while not rospy.is_shutdown():
        rospy.loginfo("我要开始刷屏了")
        msg = String()
        msg.data = "求上车+++"
        pub.publish(msg)
        rate.sleep()